PluginCode/src/main/java/smr/cs/ualberta/libcomp/action/ReplacementAction.java

```diff
@@ -141,14 +148,15 @@ public void detectAllOpenEditors() throws IOException {
 
             if (psiFile != null) {
                 FileType fileType = psiFile.getFileType();
+                String fileName = psiFile.getName();
                 String fileExtention = fileType.getDefaultExtension();
 
                 if (fileExtention.equalsIgnoreCase("java"))
-                {detectImports(psiFile, editor, project_name); }
-                if (fileExtention.equalsIgnoreCase("xml"))
-                {detectMaven(editor, psiFile, project_name);}
+                {detectImports(psiFile, editor, projectName); }
+                if (fileName.equalsIgnoreCase("pom.xml"))
+                {detectMaven(editor, psiFile, projectName);}
                 if (fileExtention.equalsIgnoreCase("groovy"))
-                {detectDependancy(editor, psiFile, project_name);}
+                {detectDependancy(editor, psiFile, projectName);}
```

Comment:
should this be called detectGradle for consistency with detectImports and detectMaven?

Reply:
sounds good